docs/website/docs/tutorial/grouping-resources.md

```diff
@@ -22,17 +25,11 @@ In the previous tutorial, we loaded issues from the GitHub API. Now we'll prepar
 def get_comments(
     updated_at = dlt.sources.incremental("updated_at", initial_value="1970-01-01T00:00:00Z")
 ):
-    url = "https://api.github.com/repos/dlt-hub/dlt/comments?per_page=100"
-
-    while True:
-        response = requests.get(url)
-        response.raise_for_status()
-        yield response.json()
-
-        # get next page
-        if "next" not in response.links:
-            break
-        url = response.links["next"]["url"]
+    for page in paginate(
+        "https://api.github.com/repos/dlt-hub/dlt/comments"
+        params={"per_page": 100}
+    ):
+        yield page
```

Comment:
I would keep one "pure python" example, so people could understand that they also can use just python if they want.
I would mention below that we have a rest api helper, we recommend to use it to save your time and build more reliable api source

Reply:
You probably mean "Requests", right? We have those examples in the first part of tutorial. I was just thinking it would be strange to return back to the hands-on Requests pagination after we switch to `paginate()'

Reply:
So the pure-requests examples are here https://deploy-preview-1287--dlt-hub-docs.netlify.app/docs/tutorial/load-data-from-an-api#load-only-new-data-incremental-loading repeating multiple times until "Using pagination helper" section.

Reply:
right!